.buildkite/docker/docker-compose.yaml

```diff
@@ -0,0 +1,51 @@
+version: '3.5'
+
+services:
+  cassandra:
+    image: cassandra:3.11
+    logging:
+      driver: none
+    ports:
+      - '9042:9042'
+
+  temporal:
+    image: temporalio/auto-setup:1.11.0
+    logging:
+      driver: none
+    ports:
+      - '7233:7233'
+      - '7234:7234'
+      - '7235:7235'
+      - '7239:7239'
+      - '6933:6933'
+      - '6934:6934'
+      - '6935:6935'
+      - '6939:6939'
```

Comment:
Not required for CI.
Instead of duplicating the entire file, why don't you download the original?
```docker-compose -f docker-compose.yml -f server.yml up```

Reply:
What original are you referring to here?

Reply:
https://github.com/temporalio/docker-compose/blob/main/docker-compose.yml